10.47 fatal symbol referencing errors on Solaris 11.4

Getting Error in make step for PCRE2 10.47.   

Undefined                       first referenced
 symbol                             in file
_lib_version                        /usr/lib/sparcv9/values-Xa.o  (symbol has no version assigned)
ld: fatal: symbol referencing errors
make[1]: *** [Makefile:2016: libpcre2-8.la] Error 1


Configure  
         export CFLAGS="-std=gnu99 -m64 -fPIC -D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64  "
         export LDFLAGS="-m64"
         ./configure  --prefix=/usr/local --libdir=/usr/local/lib/sparcv9 --enable-pcre2-8  --disable-static 

Platform Solaris 11.4 Sparc. 

I previously successfully built 10.44 on this same server, and reverted back to it, as I could not find a workaround.  10.44 does not get this error.

I tried multiple things switched from sun to gnu ld - same error.

I recompiled 10.44 today with the same script to confirm, and there is no error.     so this was introduced between 10.44 and 10,.47

--- a/src/libpcre2-16.sym
+++ b/src/libpcre2-16.sym
@@ -83,6 +83,7 @@ PCRE2_10.47 {
   local:
     _fini;
     _init;
+    _lib_version;
 };
 
 # PCRE2_10.48 {} PCRE2_10.47;

--- a/src/libpcre2-32.sym
+++ b/src/libpcre2-32.sym
@@ -83,6 +83,7 @@ PCRE2_10.47 {
   local:
     _fini;
     _init;
+    _lib_version;
 };
 
 # PCRE2_10.48 {} PCRE2_10.47;

--- a/src/libpcre2-8.sym
+++ b/src/libpcre2-8.sym
@@ -83,6 +83,7 @@ PCRE2_10.47 {
   local:
     _fini;
     _init;
+    _lib_version;
 };
 
 # PCRE2_10.48 {} PCRE2_10.47;

--- a/src/libpcre2-posix.sym
+++ b/src/libpcre2-posix.sym
@@ -8,6 +8,7 @@ PCRE2_10.47 {
   local:
     _fini;
     _init;
+    _lib_version;
 };
 
 # PCRE2_10.48 {} PCRE2_10.47;

